<launch>

    <!-- Launching the crane controller node -->
    <node name="wasp_crane_controller" pkg="extrusion_controller" type="wasp_crane_controller.py" output="screen"/>
        <!-- Setting parameters -->
        <param name="slow_motion_speed" value="0.5" />
        <param name="fast_motion_speed" value="1.0" />
        <param name="slow_extrusion_speed" value="0.4" />
        <param name="fast_extrusion_speed" value="0.8" />
        <param name="default_width" value="0.3" />
        <param name="width_threshold" value="0.05" />
        <param name="default_motion_speed" value="70" />
        <param name="default_extrusion_speed" value="120" />

</launch>
